src/convertToObject.js

```diff
@@ -7,6 +7,17 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+
+  const splitedTab = sourceString.split(';').filter(pom => /\S/.test(pom));
```

Comment:
you dont need this filter, just add condition in your loop, like if (key.trim()) {}